cmd/registry/cmd/compute/conformance.go

```diff
@@ -241,13 +305,19 @@ func (task *computeConformanceTask) Run(ctx context.Context) error {
 
 		// Lint the directory containing the spec.
 		lintProblems, err := linter.LintSpec(task.spec.GetMimeType(), filePath)
+
+		// If a linter returned an error, we shouldn't stop linting completely and discard
+		// the conformance report for this spec. We should log but still continue, because there
+		// may still be useful information from other linters that we may be discarding.
 		if err != nil {
-			return err
+			log.Log.Errorf(
+				"Linting the spec %s with the linter %s failed %s: %s",
+				task.spec.GetName(),
+				linter.GetName(),
+				err,
```

Comment:
I think we're missing an argument here. When I run the example I get this error:

 ERROR[0001] Linting the spec projects/protos/locations/global/apis/google-storage/versions/v1/specs/protos.zip with the linter api-linter failed exec: "svn": executable file not found in $PATH: %!s(MISSING) uid=[ 16fb7f3a ] 

I'm also wondering why something is trying to run `svn`... subversion?